testdata/e2e/cost_test.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cost.example.com
+spec:
+  group: example.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                bigArray:
+                  type: array
+                  maxItems: 50
+                  items:
+                    type: string
+                    maxLength: 500
+                  x-kubernetes-validations:
+                    - rule: "self.all(x, self.all(y, self.all(z, x == y && y == z)))"
```

Comment:
same here

Reply:
Marking as fixed (see [this comment](https://github.com/DangerOnTheRanger/celvet/pull/2#discussion_r965016205))